src/SelectBox/SelectBox.js

```diff
@@ -0,0 +1,79 @@
+import React, { useRef, useEffect } from 'react';
+import propTypes from 'prop-types';
+import {
+  StyledOptionsList,
+  StyledOptionsListItemInnerButton,
+  StyledOptionsListItem
+} from './SelectBox.styles';
+import { StyledCutout } from '../Cutout/Cutout';
+
+const SelectBox = React.forwardRef(function SelectBox(props) {
+  const { options, value, onSelect, width, height } = props;
+  const selectedListItemRef = useRef(null);
+  const listRef = useRef(null);
+
+  const handleKeyDown = event => {
+    const { key } = event;
+    switch (key) {
+      case 'ArrowDown':
+        if (value < options.length - 1) onSelect(value + 1);
+        break;
+      case 'ArrowUp':
+        if (value > 0) onSelect(value - 1);
+        break;
+      default:
+        break;
+    }
+  };
+
+  useEffect(() => {
+    selectedListItemRef.current.scrollIntoView({
+      behavior: 'smooth',
+      block: 'start'
+    });
+  }, [selectedListItemRef]);
+
+  return (
+    <StyledCutout>
+      <StyledOptionsList
+        style={{ width, height }}
+        ref={listRef}
+        onKeyDown={handleKeyDown}
+      >
+        {options.map(o => (
```

Comment:
Please give better names to variables instead of a single letter, in this case, we could have something like `option` or you can just destruct the properties and use them inside the loop.

What do you think?

Reply:
You're right, those single char variables doesn't mean anything.. I've never thought about this destruct way, i will try to apply it.